### ISSUE TITLE
Craft 4 Support

### ISSUE THREAD

**Issue by Floriswijgergangs** [MultiSafepay/craft-commerce#4]
Support for Craft 4 en Commerce 4.

**Comment by J0risS** [comment]
Hi @Floriswijgergangs,

Currently we are unsure wether or not we are going to support Craft 4. Once we have some news about this we will let you know.

**Comment by Floriswijgergangs** [comment]
Hi Joris, 

We're already working on Craft 4 version of your plugin. Almost finished.

**Comment by karensg** [comment]
Hey guys, did you manage to get it ready fro Craft 4?

**Comment by Floriswijgergangs** [comment]
Yes we have, but MultiSafepay is not sure to support Craft 4. In PR our version for Craft 4.
https://github.com/MultiSafepay/craft-commerce/pull/5
